### DISCUSSION TITLE
关于替换计算机学院的英文封面

### DISCUSSION THREAD

**Discussion by zyklovecoding** [whutug/whu-thesis#300] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/39197a87-eb91-46cf-a3ea-8b844dc92693)
您好，请问如何将硕士论文的英文封面替换为这个模板呢，感谢

**Comment by SwitWu** [comment, marked as answer]
在导言区使用 `\whumodule{cs}`

**Reply by zyklovecoding** [reply]
感谢您的回复
